rust-toolchain

```diff
@@ -0,0 +1 @@
+1.70.0
```

Comment:
is this a valid format?

Reply:
Oh this is the old format, let me change that 😅